lib/Widget/evaluationForms/post-hos/post-hos-day0/recovery_readiness_form.dart

```diff
@@ -0,0 +1,376 @@
+                                              _vomit = value;
+                                              _readyRecover =
+                                                  completeAllCheckBox();
+                                            });
+                                          },
+                                          title: Text(
+                                              '8. ไม่มีอาการคลื่นไส้ อาเจียน'),
+                                        ),
+                                      ],
+                                    ),
+                                  ),
+                                  Padding(
+                                    padding: const EdgeInsets.all(8.0),
+                                    child: Container(
+                                      child: RaisedButton(
+                                          shape: RoundedRectangleBorder(
+                                              borderRadius:
+                                                  BorderRadius.circular(7.0)),
+                                          textColor: Colors.white,
+                                          padding: EdgeInsets.all(15),
+                                          color: Color(0xFF2ED47A),
+                                          child: Text('ยืนยัน',
+                                              style: TextStyle(fontSize: 18)),
+                                          onPressed: () async {
+                                            if (_knowFullName |
+                                                    _knowTime |
+                                                    _knowPlace |
+                                                    _fever |
+                                                    _breathGasp |
+                                                    _palpitations |
+                                                    _dizzy |
+                                                    _vomit !=
+                                                true) {
+                                              Dialogs
+                                                  .alertToCompleteEvalutation(
+                                                      context);
+                                            } else {
+                                              Map<String, dynamic>
+                                                  formDataToDB = {
+                                                'Choice1': _knowFullName,
+                                                'Choice2': _knowTime,
+                                                'Choice3': _knowPlace,
+                                                'Choice4': _fever,
+                                                'Choice5': _breathGasp,
+                                                'Choice6': _palpitations,
+                                                'Choice7': _dizzy,
+                                                'Choice8': _vomit,
```

Comment:
I think we should name the fields as well

Reply:
Solved in aaa4fed kub